src/fastga_he/models/propulsion/components/loads/sm_pmsm/components/sizing_active_length.py

```diff
@@ -0,0 +1,67 @@
+# This file is part of FAST-OAD_CS23-HE : A framework for rapid Overall Aircraft Design of Hybrid
+# Electric Aircraft.
+# Copyright (C) 2025 ISAE-SUPAERO
+
+import numpy as np
+import openmdao.api as om
+
+
+class SizingActiveLength(om.ExplicitComponent):
+    """
+    Computation of the length in the SM PMSM that is electromagnetically active. The formula is
+    obtained from equation (II-44) in :cite:`touhami:2020`.
```

Comment:
```suggestion
    taken from equation (II-44) in :cite:`touhami:2020`.
```
I would you only use obtained if you modified the formula